pkg/network/driver_interface.go

```diff
@@ -50,13 +53,27 @@ var (
 // DriverExpvar is the name of a top-level driver expvar returned from GetStats
 type DriverExpvar string
 
+// This is the type that an overlapped read returns -- the overlapped object, which must be passed back to the kernel after reading
+// followed by a predictably sized chunk of bytes
+type _readbuffer struct {
```

Comment:
Is the `_` prefix a convention we use elsewhere? What is it supposed to signify?

Reply:
This is copied from some sample code, I can change it to be just `readbuffer`